config/service_alias.go

```diff
@@ -22,9 +22,49 @@ const (
 // gateway.QoSService interface.
 // THis is to handle requests for a given service ID.
 var ServiceQoSTypes = map[protocol.ServiceID]ServiceQoSType{
-	// TODO_IMPROVE(@commoddity): Add all EVM service IDs here.
+	// All Morse EVM Services as of 12/17/2024
+	"F001": ServiceIDEVM, // Arbitrum One
+	"F002": ServiceIDEVM, // Arbitrum Sepolia Testnet
+	"F003": ServiceIDEVM, // Avalanche
+	"F004": ServiceIDEVM, // Avalanche-DFK
+	"F005": ServiceIDEVM, // Base
+	"F006": ServiceIDEVM, // Base Sepolia Testnet
+	"F008": ServiceIDEVM, // Blast
+	"F009": ServiceIDEVM, // BNB Smart Chain
+	"F00A": ServiceIDEVM, // Boba
+	"F00B": ServiceIDEVM, // Celo
+	"F00C": ServiceIDEVM, // Ethereum
+	"F00D": ServiceIDEVM, // Ethereum Holesky Testnet
+	"F00E": ServiceIDEVM, // Ethereum Sepolia Testnet
+	"F00F": ServiceIDEVM, // Evmos
+	"F010": ServiceIDEVM, // Fantom
+	"F011": ServiceIDEVM, // Fraxtal
+	"F012": ServiceIDEVM, // Fuse
+	"F013": ServiceIDEVM, // Gnosis
+	"F014": ServiceIDEVM, // Harmony-0
+	"F015": ServiceIDEVM, // IoTeX
+	"F016": ServiceIDEVM, // Kaia
+	"F017": ServiceIDEVM, // Kava
+	"F018": ServiceIDEVM, // Metis
+	"F019": ServiceIDEVM, // Moonbeam
+	"F01A": ServiceIDEVM, // Moonriver
+	"F01C": ServiceIDEVM, // Oasys
+	"F01D": ServiceIDEVM, // Optimism
+	"F01E": ServiceIDEVM, // Optimism Sepolia Testnet
+	"F01F": ServiceIDEVM, // opBNB
+	"F021": ServiceIDEVM, // Polygon
+	"F022": ServiceIDEVM, // Polygon Amoy Testnet
+	"F024": ServiceIDEVM, // Scroll
+	"F027": ServiceIDEVM, // Taiko
+	"F028": ServiceIDEVM, // Taiko Hekla Testnet
+	"F029": ServiceIDEVM, // Polygon zkEVM
+	"F02A": ServiceIDEVM, // zkLink
+	"F02B": ServiceIDEVM, // zkSync
+
+	// TODO_IMPROVE Add all non-EVM Morse Services and requisite initialization
+
+	// Other Services (Shannon / Testing)
 	"0021":        ServiceIDEVM, // ETH Mainnet
-	"F00C":        ServiceIDEVM, // ETH Mainnet (Full/Archival)
 	"anvil":       ServiceIDEVM, // ETH Local (development/testing)
 	"eth":         ServiceIDEVM, // ETH general-purpose catch-all
```

Comment:
Same for `eth` and `eth-mainnet` since these are aliases not "authoritative service IDs" (new term coming in hot).